app/main.py

```diff
@@ -40,10 +41,32 @@ def create_tables(engine, psql_environment):
 set_ui_language()
 
 from app.routers import (  # noqa: E402
-    about_us, agenda, calendar, categories, celebrity, credits,
-    currency, dayview, email, event, export, four_o_four, friendview,
-    google_connect, invitation, joke, login, logout, profile,
-    register, search, telegram, user, weekview, weight, whatsapp,
+    about_us,
```

Comment:
Maybe revert?

Reply:
The pre commit script changed it, I am not sure we need to revert.